assistant/v1.ts

```diff
@@ -5336,15 +5336,15 @@ namespace AssistantV1 {
     message_to_human_agent?: string;
     /** The text of the search query. This can be either a natural-language query or a query that uses the Discovery
      *  query language syntax, depending on the value of the **query_type** property. For more information, see the
-     *  [Discovery service documentation](https://cloud.ibm.com/docs/discovery/query-operators.html#query-operators).
+     *  [Discovery service documentation](https://cloud.ibm.com/docs/discovery?topic=discovery-query-operators#query-operators).
```

Comment:
Is this being addressed in the API Definition or does it need to be documented as a hand-written change?

Reply:
nope already addressed